Formatting

thehappydinoa opened the pull request:
- PEP8 Format
- Organized imports
- Added link to `README.md`
- Formatted html and css

thehappydinoa committed:
PEP8 Formatting

batuhaniskr commented on tracking.py:
```diff
@@ -130,25 +143,25 @@ def receive_buffer(tweets):
-            |   |   |  \  | \  ___/|  |_|  |_|  / /_/  >  ___/|   |  \  \__\  ___/ 
+    print(colored('''\n\t\t\033[1m
+             ___________       .__.__  __
+             \__    ___/_  _  _|__|__|/  |_  ___________
```
Tool Logo is incorrected format when running tracking.py. You can look https://ibb.co/cN0hfV link.

thehappydinoa replied:
Ah let me check that out.

thehappydinoa replied:
Fixed

thehappydinoa committed:
Fixed logo

batuhaniskr commented on analysis.py:
```diff
@@ -62,7 +66,8 @@ def analysis_user():
         c = conn.cursor()
-        c.execute("SELECT  username, count(*) as tekrar FROM Tweet  group by username order by tekrar desc LIMIT 10")
+        c.execute(
+            "SELECT  username, count(*) as tekrar FROM Tweet  group by username order by tekrar desc LIMIT 10")
```
Why was it add blank line in c.execute?

batuhaniskr commented on analysis.py:
```diff
@@ -53,7 +56,8 @@ def main(argv):
     api_key = settings.GOOGLE_MAP_API_KEY
-    url = 'https://maps.googleapis.com/maps/api/js?key=' + api_key + '&libraries=visualization&callback=initMap'
+    url = 'https://maps.googleapis.com/maps/api/js?key=' + \
+        api_key + '&libraries=visualization&callback=initMap'
```
There is also a blank line here.

batuhaniskr commented on gui/socialgui.py:
```diff
@@ -64,15 +62,18 @@ def setupUi(self, So):
         font.setWeight(75)
         self.pushButton_2.setFont(font)
-        self.pushButton_2.setStyleSheet(_fromUtf8("background-color:rgb(65, 65, 65); color:rgb(255, 255, 255);"))
+        self.pushButton_2.setStyleSheet(
```
+

thehappydinoa commented:
This is just PEP8 formatting.

thehappydinoa committed:
Merge branch 'master' into master

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 19.30kb -> 16.38kb (15.15%)

/images/analysisn.png -- 17.00kb -> 14.09kb (17.11%)
/images/logo.png -- 2.29kb -> 2.28kb (0.6%)

thehappydinoa committed:
Merge pull request #1 from thehappydinoa/imgbot

[ImgBot] Optimize images

Co-authored-by: ImgBot <ImgBotHelp@gmail.com>
